### PR TITLE
en, zh: bump TiDB Operator charts version to v1.2.0-beta.1

### PR DESCRIPTION
**Pull request by handlerww**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)
bump tidb operator charts version to v1.2.0-beta.1

```bash
find . -type f | xargs sed -i "s/v1.2.0-alpha.1/v1.2.0-beta.1/g"
```

ref #1050

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 
https://github.com/pingcap/tidb-operator/issues/3704
### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/releases/release-1.2.0-alpha.1.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ TiDB Operator version: 1.2.0-alpha.1
 
 ## Rolling Update Changes
 
-- Due to [#3440](https://github.com/pingcap/tidb-operator/pull/3440), the Pod of TidbMonitor will be deleted and recreated after TiDB Operator is upgraded to v1.2.0-alpha.1.
+- Due to [#3440](https://github.com/pingcap/tidb-operator/pull/3440), the Pod of TidbMonitor will be deleted and recreated after TiDB Operator is upgraded to v1.2.0-beta.1.
```
Comment (csuzhangxc) [inline en/releases/release-1.2.0-alpha.1.md:13]: I think we should not update this previous release note.

File: zh/releases/release-1.2.0-alpha.1.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ TiDB Operator 版本：1.2.0-alpha.1
 
 ## 滚动升级改动
 
-- 由于 [#3440](https://github.com/pingcap/tidb-operator/pull/3440) 的改动，升级 TiDB Operator 到 v1.2.0-alpha.1 版本后，TidbMonitor Pod 会被删除重建。
+- 由于 [#3440](https://github.com/pingcap/tidb-operator/pull/3440) 的改动，升级 TiDB Operator 到 v1.2.0-beta.1 版本后，TidbMonitor Pod 会被删除重建。
```
Comment (csuzhangxc) [inline zh/releases/release-1.2.0-alpha.1.md:13]: ditto

File: en/releases/release-1.2.0-alpha.1.md (hunk 1 above)
Comment (ran-huang) [inline en/releases/release-1.2.0-alpha.1.md:13]: Please revert this.

File: zh/releases/release-1.2.0-alpha.1.md (hunk 2 above)
Comment (ran-huang) [inline zh/releases/release-1.2.0-alpha.1.md:13]: Please revert this.

**Review by ran-huang — APPROVED** [review]
Rest LGTM

File: zh/releases/release-1.2.0-alpha.1.md (hunk 2 above)
Comment (handlerww) [reply zh/releases/release-1.2.0-alpha.1.md]: done

Comment (handlerww) [reply zh/releases/release-1.2.0-alpha.1.md]: fixed

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Pull request merged**